scripts/wca-bootstrap.sh

```diff
@@ -36,6 +38,14 @@ fi
 if ! command -v gcc &> /dev/null; then
   apt-get install -y build-essential
 fi
+if ! command -v zip &> /dev/null; then
+apt-get install -y zip
+fi
+if ! command -v aws &> /dev/null; then
+curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"
+unzip awscliv2.zip
+sudo ./aws/install
```

Comment:
Is this the Amazon-recommended way of installing AWS CLI?

Reply:
For the record: Yes, it is: https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html